otter/plugins/builtin/grade_override.py

```diff
@@ -110,11 +110,11 @@ def during_generate(self, otter_config, assignment):
             curr_dir = os.getcwd()
             os.chdir(assignment.master.parent)
 
-        cfg_idx = [self.IMPORTABLE_NAME in c.keys() for c in otter_config["plugins"] if isinstance(c, dict)].index(True)
-        creds_path = otter_config["plugins"][cfg_idx][self.IMPORTABLE_NAME]["credentials_json_path"]
+        cfg_idx = [self.IMPORTABLE_NAME in c.keys() for c in assignment.plugins if isinstance(c, dict)].index(True)
```

Comment:
The `assignment` variable will be `None` in the case where Otter Generate is used without Otter Assign; in that case, `otter_config` is what needs to be updated. Maybe create a new variable `plugins = assignment.plugins if assignment else otter_config["plugins"]` and use that in the rest of the function.

Reply:
Got it. Just pushed it in